gm4_vertical_rails/data/vertical_rails/functions/momentum.mcfunction

```diff
@@ -0,0 +1,5 @@
+execute if block ~ ~ ~ ladder run tp ~ ~.3 ~
```

Comment:
At this point, the function should only be running if the minecart is in a ladder. The teleport should not double-check the block.